src/languages/es-ES.js

```diff
@@ -0,0 +1,2 @@
+export default {
```

Comment:
Why doesn't this one have `hello: 'Hola',` ?

Reply:
As `es-ES` and `es` have the same translation for this word, I thought it's better to use this word from `es`. I thought it would make more sense to add words with different translations for `es-ES` locale here.